dspy/adapters/chat_adapter.py

```diff
@@ -67,6 +67,23 @@ def parse(self, signature, completion, _parse_values=True):
 
         return fields
 
+    def format_completion(self, signature, outputs):
```

Comment:
What's the goal here? It creates a lot of logic duplication, i.e. editing the format now requires editing one more place.

Reply:
Used to generate prompt/completion pair given a trace.

Is there a better way to do it?

Reply:
Yes, discussed offline